### ISSUE TITLE
Disconnection without warning

### ISSUE THREAD

**Issue by digifelis** [StamusNetworks/SELKS#194]
The page at https://hs01.kep.tr opens, but not all items are loaded. the tabs in the page remain inactive. without warning or alarm. There are no rules for this page.

**Comment by pevma** [comment]
Can you please share a screenshot of the problem ?

**Comment by digifelis** [comment]
![kep](https://user-images.githubusercontent.com/5615913/62627846-d7b55780-b932-11e9-95ad-e52b70e3c5b6.png)

I cant click SMS tap at this page. SMS tap is openning at normal. 

I write the following rules. than problem over.

pass tcp any any -> 95.0.68.26 any (msg:"kep iletisim1"; sid:5000001; noalert; bypass;)
pass udp any any -> 95.0.68.26 any (msg:"kep iletisim2"; sid:5000002; noalert; bypass;)
pass http any any -> 95.0.68.26 any (msg:"kep iletisim3"; sid:5000003; noalert; bypass;)
pass tcp 95.0.68.26 any -> any any (msg:"kep iletisim4"; sid:5000004; noalert; bypass;)
pass udp 95.0.68.26 any -> any any (msg:"kep iletisim5"; sid:5000005; noalert; bypass;)
pass http 95.0.68.26 any -> any any (msg:"kep iletisim6"; sid:5000006; noalert; bypass;)

**Comment by pevma** [comment]
Are using IPS or IDS mode ?

**Comment by digifelis** [comment]
IPS mode.

**Comment by pevma** [comment]
It seems some rules are stopping from opening the page you want - unless you explicitly use the `pass` rules as you mentioned. 
If you want to understand which rules are  preventing the page to be opened maybe you cab have a look at the `drop` action/logs in eve.json when you open the page.

**Comment by digifelis** [comment]
yes, as you say i inspected eve.json . but i cant find anything. so i check evebox panel but there are no warnings.
now i deleted all default rules. but problem is continues.

**Comment by pevma** [comment]
So no alerts in eve,json at all ?

**Comment by digifelis** [comment]
no alerts in eve.json

**Comment by pevma** [comment]
What is the output of `ls -lh /var/log/suricata/` ?
